ui/app/templates/components/wizard-progress.hbs

```diff
@@ -1,5 +1,6 @@
 <div class="progress-container">
   {{#each progressBar as |bar|}}
+  {{log bar "bar ARG TODO COME BACK TO ME"}}
```

Comment:
remove

Reply:
Oish, thank you.  There is a progress bar issue.  I'll work on this.